### DISCUSSION TITLE
Any TerraOS prebuilt images to download?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1071] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I don't have a way to build the TerraOS image so I can flash it. Does any body have a download page for a prebuilt one?

**Comment by leoisbadatmakeingthings** [comment, marked as answer]
https://files.mercurywork.shop/r58playz/terraos/
pick your board if it isn't listed it won't work. The ChromeOS and arch image is broken the arch partition doesn't boot so image the one from the arch only image and restore it to the arch partition on the chromeos and arch image.
